.github/workflows/style-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Vale Lint (Changed Markdown Only)
+
+on:
+  pull_request:
+    paths:
+      - '**/*.md'
```

Comment:
I noticed there are a lot of errors reported by vale. Ideally we should enable this once all these are fixed, but since that's impractical, shall we enforce this to a subfoder with zero vale errors and then gradually expand the range as we fix the rest of the files?